src/video/emscripten/SDL_emscriptenevents.c

```diff
@@ -409,7 +409,25 @@ static EM_BOOL
 Emscripten_HandleWheel(int eventType, const EmscriptenWheelEvent *wheelEvent, void *userData)
 {
     SDL_WindowData *window_data = userData;
-    SDL_SendMouseWheel(window_data->window, 0, (float)wheelEvent->deltaX, (float)-wheelEvent->deltaY, SDL_MOUSEWHEEL_NORMAL);
+
+    float deltaY = wheelEvent->deltaY;
+
+    switch (wheelEvent->deltaMode) {
+        case 0:
```

Comment:
Why not actually use `DOM_DELTA_PIXEL` et al from `emscripten/html5.h` here in the switch?